.github/workflows/doctor.yml

```diff
@@ -18,13 +18,13 @@ jobs:
     strategy:
       matrix:
         include:
-          - runner: "13-arm64-${{github.run_id}}-${{github.run_attempt}}"
-          - runner: "12-arm64-${{github.run_id}}-${{github.run_attempt}}"
-          - runner: "12-${{github.run_id}}-${{github.run_attempt}}"
+          - runner: "13-arm64-${{ github.run_id }}"
+          - runner: "12-arm64-${{ github.run_id }}"
+          - runner: "12-${{ github.run_id }}"
           - runner: "11-arm64"
             cleanup: true
-          - runner: "11-${{github.run_id}}-${{github.run_attempt}}"
-          - runner: "10.15-${{github.run_id}}-${{github.run_attempt}}"
+          - runner: "11-${{ github.run_id }}"
+          - runner: "10.15-${{ github.run_id }}"
```

Comment:
Does this even still work?

Reply:
It likely does. The image still exists on our Orka cluster and the mapping still exists in ci-orchestrator - there's not really been a need to remove it.

Though we probably should drop it from here anyway.

Reply:
So it does. It needs to be replaced with an Intel Ventura run anyway.

Reply:
...wait, shouldn't `brew doctor` fail on 10.15 now? With some sort of `unsupported macOS` error.

Reply:
`HOMEBREW_DEVELOPER=1` actually skips that check.

Reply:
Removed.

Will give this a re-run to see how the orchestrator handles it after.

Reply:
Good idea :+1:

Reply:
Rerun attempted... https://github.com/Homebrew/brew/actions/runs/5112934741/jobs/9191889634 (only Intel Monterey)

Reply:
That works, I think?

<img width="643" alt="Screenshot 2023-05-29 at 23 01 42" src="https://github.com/Homebrew/brew/assets/30379873/a76f953a-0aaa-4a32-8124-4da428edfdbb">

Reply:
LGTM

Reply:
Also now running. Nice. Let's try re-running all of them...